cob_bringup/tools/android.launch

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
```

Comment:
instead or renaming it we should check if we can get rid of it completly, see https://github.com/ipa320/cob_android/issues/5.